src/utils/filterutils.tsx

```diff
@@ -43,6 +43,26 @@ export function isDuplicateFieldValueFilter(
   return true;
 }
 
+/**
+ * Returns true if the two given static filters are the same.
```

Comment:
What are cases that this happens? Is it for checking if two variables reference the same filter or for catching when the same field is being filtered on by two separate instances of the StaticFilter?

Reply:
This is for the former (two variables referencing the same filter)